sci-libs/libsigrokdecode/libsigrokdecode-0.5.3.recipe

```diff
@@ -41,18 +43,28 @@ REQUIRES_devel="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
+	devel:libcheck$secondaryArchSuffix	# for tests to actually run.
```

Comment:
If this is just needed for tests, this should go into `TEST_REQUIRES` (`cmd:cmp` and `cmd:diff` too?)

Reply:
Do the test fail if you don't have them listed in BUILD_PREREQUIRES?

Reply:
OK, if the tests aren't built at all when not including libcheck, I think it's better to keep it in `BUILD_REQUIRES`.